File: examples/cone.cpp
#include <string>
#include <sstream>
#include <unistd.h>
#include <iostream>

#include "hilma/types/Mesh.h"
#include "hilma/ops/generate.h"
#include "hilma/io/ply.h"

int main(int argc, char **argv) {

    hilma::Mesh mesh = hilma::cone(1.f, 2.f, 36, 1, 1);
    hilma::savePly("cone.ply", mesh, false);

    return 1;
}

File: examples/cone.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
from __future__ import unicode_literals

from hilma import Mesh, cone, savePly

mesh = cone(1., 2., 36, 1, 1)
savePly("cone.ply", mesh, False)
